onos-tost/values.yaml

```diff
@@ -0,0 +1,26 @@
+onos-classic:
```

Comment:
Can we show an example of changing replica and atomix.replica here - it can be very useful if people want to deploy single instance ONOS